packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -19,6 +24,76 @@ export function Widget({ starterVariant }) {
     (state) => state.onOpenNavbarWalletSelectMenu,
   );
 
+  const handleMultiSigTransactionDetails = async (txHash: string) => {
+    const safeProviderSDK = (account?.signer?.provider as any)?.provider?.sdk;
+
+    const safeTransactionDetails: GatewayTransactionDetails =
+      await safeProviderSDK.txs.getBySafeTxHash(txHash);
+
+    if (safeTransactionDetails.txStatus === TransactionStatus.SUCCESS) {
+      return {
+        status: 'SUCCESS',
```

Comment:
Maybe we should use 'DONE' here, as we always use DONE instead of SUCCESS in the SDK.